distribution/docs/src/main/resources/_contents/_data/media-attributes-table-contents.adoc

```diff
@@ -72,18 +72,20 @@ STANAG 4559 NC, NM, C1, M1, I1, C3, M3, C4, M4, C5, M5, C8, M8
 |A two-part identifier for file formats and format content
 |String
 |a valid https://en.wikipedia.org/wiki/Media_type[mime-type]
+TODO mime-type is defined in SIX linked RFC specs, but wikipedia isn't really authoritative
```

Comment:
Agree.  Maybe just link to the RFC directly?  https://www.ietf.org/rfc/rfc2046.txt